gallery/gallery/lib/l10n/messages_all.dart

```diff
@@ -15,485 +15,485 @@ import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 import 'package:intl/src/intl_helpers.dart';
 
-import 'messages_af.dart' as messages_af;
-import 'messages_am.dart' as messages_am;
-import 'messages_ar.dart' as messages_ar;
-import 'messages_ar_EG.dart' as messages_ar_eg;
-import 'messages_ar_JO.dart' as messages_ar_jo;
-import 'messages_ar_MA.dart' as messages_ar_ma;
-import 'messages_ar_SA.dart' as messages_ar_sa;
-import 'messages_as.dart' as messages_as;
-import 'messages_az.dart' as messages_az;
-import 'messages_be.dart' as messages_be;
-import 'messages_bg.dart' as messages_bg;
-import 'messages_bn.dart' as messages_bn;
-import 'messages_bs.dart' as messages_bs;
-import 'messages_ca.dart' as messages_ca;
-import 'messages_cs.dart' as messages_cs;
-import 'messages_da.dart' as messages_da;
-import 'messages_de.dart' as messages_de;
-import 'messages_de_AT.dart' as messages_de_at;
-import 'messages_de_CH.dart' as messages_de_ch;
-import 'messages_el.dart' as messages_el;
-import 'messages_en_AU.dart' as messages_en_au;
-import 'messages_en_CA.dart' as messages_en_ca;
-import 'messages_en_GB.dart' as messages_en_gb;
-import 'messages_en_IE.dart' as messages_en_ie;
-import 'messages_en_IN.dart' as messages_en_in;
-import 'messages_en_NZ.dart' as messages_en_nz;
-import 'messages_en_SG.dart' as messages_en_sg;
-import 'messages_en_US.dart' as messages_en_us;
-import 'messages_en_ZA.dart' as messages_en_za;
+import 'messages_ko.dart' as messages_ko;
```

Comment:
@shihaohong do you know why the latest version of the l10n script moved all these around?

Reply:
This was a regression when I introduced tests, and I made a fix immediately after. 

https://github.com/flutter/flutter/pull/46247

Could you review and it approve it if it looks good?

Reply:
Thanks. I'll revert the change here then, and then we rerun this when your change has landed in flutter/flutter.

Reply:
The sorting fix has landed! Not sure if I need to let you all know here, but just in case :)